mycroft/messagebus/message.py

```diff
@@ -36,10 +55,42 @@ def serialize(self):
 
     @staticmethod
     def deserialize(value):
+        """This takes a string and constructs a message object.
+
+        This makes it easy to take strings from the websocket and create
+        a message object.  This uses json loads to get the info and generate
+        the message object.
+
+        Args:
+            value(str): This is the json string received from the websocket
+
+        Returns:
+            Message: message object constructed from the json string passed
+            int the function.
+        """
         obj = json.loads(value)
         return Message(obj.get('type'), obj.get('data'), obj.get('context'))
 
     def reply(self, type, data, context={}):
+        """This is used to construct a reply message for a give message
+
+        This will take the same parameters as a message object but use
```

Comment:
This comment is more verbose than the code itself, to the point of not being helpful (I'm actually confused by it). This comment should reflect the goal of reply (which is to respond to a target if provided, while maintaining context).